spec/takeaway_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) {described_class.new({menu_klass: menu,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.